hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/user/security/ua/UserManager.java

```diff
@@ -106,10 +117,32 @@ public void resetKey(int id) {
 
   }
 
-  public void resetPassword(Users p, String pass) {
-    p.setPassword(pass);
-    p.setPasswordChanged(new Timestamp(new Date().getTime()));
-    em.merge(p);
+  public void resetPassword(Users p, String pass) throws Exception {
```

Comment:
If the user is owner of a project then we should also update the project generic certificate's password.